File: src/cool/controller/MainController.php
<?php

namespace App\Http\Controllers;

use Illuminate\Routing\Controller as BaseController;
use Symfony\Component\HttpFoundation\Request;
use Illuminate\Support\Facades\DB;

class MainController extends BaseController
{
    public function getList1(Request $request){

        return response()->view("list1");
    }

    public function getList2(Request $request){

        return response()->view("list2");
    }

    public function getList3(Request $request){

        return response()->view("list3");
    }

    public function getList4(Request $request){

        return response()->view("list4");
    }

    public function getList5(Request $request){

        return response()->view("list5");
    }

    public function getPlay(Request $request){

        return response()->view("play");
    }

    public function getProjects(Request $request){
        $type=$request->session()->get('type',1);
        $id=$request->session()->get('id',0);
        $result2=DB::select('select * from projects where find_in_set(?,type)>0 order by id asc', [$type]);
        $curIndex=0;
        foreach ($result2 as $key=>$value){
            if($value->id==$id){
                $curIndex=$key;
                break;
            }
        }
        return response()->view("main2",[
            'data'=>$result2,
            'curIndex'=>$curIndex,
            'curRight_1'=>'project'
        ]);
    }
}

File: src/cool/controller/MainController.java
package cool.controller;

import cool.model.Video;
import cool.util.DBConnPool;
import cool.util.VideoUtil;
import org.springframework.stereotype.Controller;
import org.springframework.ui.ModelMap;
import org.springframework.web.bind.annotation.ExceptionHandler;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseBody;

import javax.servlet.http.HttpServletResponse;
import java.io.IOException;
import java.io.OutputStream;
import java.sql.Connection;
import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.util.ArrayList;
import java.util.List;


@Controller
public class MainController
{
    @RequestMapping("/list1.do")
    public String getList1(ModelMap model){
        model.put("var1",1);
        model.put("var2",2);
        model.put("var3", 3);
        return "list1";
    }

    @RequestMapping("/list2.do")
    public void getList2(HttpServletResponse response) throws IOException {
        byte[] data=VideoUtil.getVideoFrame("mv/1.mp4",2,256,256);
        response.setContentType("image/jpg");
        OutputStream stream = response.getOutputStream();
        stream.write(data);
        stream.close();
    }

    @RequestMapping("/list3.do")
    public String getList3(ModelMap model){

        return "list3";
    }

    @RequestMapping("/list4.do")
    public String getList4(ModelMap model){
        return "list4";
    }

    @RequestMapping("/list5.do")
    public String getList5(ModelMap model){
        return "list5";
    }
}
